project.clj

```diff
@@ -18,23 +18,24 @@
             :url "http://iplantcollaborative.org/sites/default/files/iPLANT-LICENSE.txt"}
   :manifest {"Git-Ref" ~(git-ref)}
   :uberjar-name "facepalm-standalone.jar"
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/tools.cli "0.3.1"]
-                 [org.clojure/tools.logging "0.3.0"]
-                 [cheshire "5.5.0"]
-                 [com.cemerick/pomegranate "0.3.0"]
-                 [fleet "0.10.1"]
-                 [com.mchange/c3p0 "0.9.5.1"]
-                 [korma "0.4.0"
+  :dependencies [[org.clojure/clojure "1.10.1"]
+                 [org.clojure/tools.cli "1.0.194"]
+                 [org.clojure/tools.logging "1.1.0"]
+                 [cheshire "5.10.0"]
+                 [clj-commons/pomegranate "1.2.0"]
+                 [fleet "0.10.2"]
+                 [com.mchange/c3p0 "0.9.5.5"]
+                 [korma "0.4.3"
                   :exclusions [c3p0]]
                  [me.raynes/fs "1.4.6"]
-                 [log4j "1.2.16"]
-                 [org.cyverse/clojure-commons "2.8.0"]
-                 [org.cyverse/kameleon "3.0.1"]
-                 [org.postgresql/postgresql "42.2.2"]
-                 [slingshot "0.10.3"]
+                 [log4j "1.2.17"]
+                 [org.cyverse/clojure-commons "3.0.5"]
+                 [org.cyverse/kameleon "3.0.4"]
+                 [org.postgresql/postgresql "42.2.14"]
+                 [slingshot "0.12.1"]
                  [clj-http "2.0.0"]]
-  :plugins [[lein-marginalia "0.7.1"]
+  :plugins [[lein-ancient "0.6.15"]
+            [lein-marginalia "0.7.1"]
             [test2junit "1.2.2"]]
```

Comment:
All changes to project.clj are just dependency updates because we haven't done any in a while.